app/controllers/TransferAgreementController.scala

```diff
@@ -22,7 +21,7 @@ class TransferAgreementController @Inject()(val controllerComponents: SecurityCo
                                             val keycloakConfiguration: KeycloakConfiguration,
                                             langs: Langs)
                                            (implicit val ec: ExecutionContext) extends TokenSecurity with I18nSupport {
-  val transferAgreementForm: Form[TransferAgreementData] = Form(
+  val emptyTransferAgreementForm: Form[TransferAgreementData] = Form(
```

Comment:
Why is this referred to as "empty". Can it not just remain named as: `transferAgreementForm`, or maybe `standardTransferAgreementForm`?

Reply:
I renamed it this for clarity, just to differentiate it from a filled in/partially filled in form, but I can change it back.